.github/workflows/test-nightly.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Defense Unicorns
```

Comment:
This workflow and corresponding task changes may overlap/conflict with https://github.com/defenseunicorns/uds-core/pull/1578 - okay merging either but maybe better to keep that separate rather than including in this PR focused on FIPS?

cc @UnicornChance - however we want to deconflict is fine, just for awareness.

Reply:
Absolutely! Let me remove this from my PR